docs/spec_draft.md

```diff
@@ -682,6 +725,47 @@ specification. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.sqrt
+
+`stablehlo.sqrt(operand) -> result`
+
+### Semantics
+
+Performs element-wise square root operation on `operand` tensor and produces a
+`result` tensor, implementing the `squareRoot` operation from the IEEE-754
+specification.
```

Comment:
I noticed that `rsqrt` says "Numeric precision is implementation-defined", but `sqrt` doesn't. Why is there a difference?

Reply:
`squareRoot` is defined as an arithmetic operation in section 5.4.1 (required to have 0.5 ulp) whereas `rSqrt` in Table 9.1 is *recommended* to have correctly rounded functions (this combines two arithmetic operations `division` and `squareRoot` which does not guarantee 0.5 ulp when combined together), thus having to state so for `rsqrt`.

Reply:
IMO, for `rsqrt` op, the IEEE standard delegates the precision to be defined by the language standard or implementation. For example, if implemented in a naive way, `sqrt` followed by `divide`, can have two rounding errors as opposed to one if implemented using very large precision computation.

OTOH, for ops like `sqrt` the precision is defined as

```
In this standard, a computational operation that returns a numeric result first produces an unrounded result
as an exact number of infinite precision. That unrounded result is then rounded to a destination format.

```
 
I think that is the reason `rsqrt` have its precision documented in this spec as implementation defined.

Reply:
Thank you for the detailed explanation! I didn't realize this.